subcommittees/proposal_instructions.md

```diff
@@ -0,0 +1,35 @@
+
+Remember, a modest **budget will be available**, approved on an item-by-item basis by the Steering Committee, to help you realize the goals of your Subcommittee or Task Force, once approved, so don’t forget to factor that into your plan.
+
+### Step 3: Submit an Issue
+The Steering Committee can’t wait to hear about your ideas and support you as you make them a reality. Once you have a draft of a plan that addresses the points above (even rough notes are totally fine!), let us know by [submitting an issue here.](https://github.com/swcarpentry/board/issues)
+
+### Step 4: Proposal Review, Revision, and Approval
+The Steering Committee will provide feedback to help you refine your plan, connect with others members who might like to assist, and work with you over the long term to achieve your goals. At this point, we will determine who will be the SCF liaison that will facilitate communication between your group and the Steering Committee. The criteria that the Steering Committee will be looking for and working with you to infuse into your proposal are:
+
+ - **Is there a concrete, measurable goal?**  We will be looking for specific goals, like “organize an event to do X” or “maintain a database to let us measure Y”. 
+ - **Is the proposed timeline appropriate?** We’ll be looking for some realistic estimate of what needs to be done and how long it will take. A timeline with milestones is something that can be revisited and reevaluated throughout the year as the project progress, so it is useful even if it happens to change over the lifetime of the project.
+ - **Is there a reasonable prediction of the resources and support the project needs to succeed?** We’d like you to anticipate as many expenses and other needs as possible up front. For example, will you need a private GitHub repo, a license for a software tool, or space on the Software Carpentry website? These items should be mentioned in your proposal. 
+ - **Have you recruited enough participants to help carry out the work?**  While you may be able to execute the project, we request that a minimum of two people are willing to support the project, with one person serving as the chair. If you or the Steering Committee believe more people are needed to support the effort, we can assist in recruiting sufficient community support.  We are keen to see you have enough people-power to succeed, while keeping things focused enough that everyone gets a chance to be actively involved.
+ - **Does the proposal articulate how this project will advance Software Carpentry’s mission? Does it complement existing efforts?** As there will be a rotating lineup of Task Forces and Subcommittees, we want to make sure people with similar ideas are working together, and that every volunteer’s time is used well; the Steering Committee will help orient you and your team within the broader community of projects and volunteers, and make sure you have a great fit before getting started. 
```

Comment:
Subcommittees and Task Forces are not destined to live forever - as interest and need wanes, past projects will be retired. We don't want to get into the details of this right now, since the point of these documents is to get folks started without burying them in infinite details; also, a lot of the lifecycle of these groups will be (a) hashed out as we see how things go rather than prescribed too far in advance, and (b) communicated in large part through the Steering Committee liaison. More on this soon :)